docs/CHANGELOG.asciidoc

```diff
@@ -15,6 +15,8 @@
 
 === Enhancements
 
+Improve and use periodic boundary condition for seasonal component modeling ({pull}84[#84])
```

Comment:
This is labelled to go in 6.4 so I think the release note should also be in the 6.4 section. Still getting my head around it though :-)